containers/payments/paymentTokenHelper.js

```diff
@@ -101,7 +101,7 @@ export const process = ({ Token, api, approvalURL, secureURL, signal }) => {
         const onMessage = (event) => {
             const origin = event.origin || event.originalEvent.origin; // For Chrome, the origin property is in the event.originalEvent object.
 
-            if (origin !== secureURL) {
+            if (!origin.includes(ReturnHost)) {
```

Comment:
Did you use `includes` because the `ReturnHost` does not contain `https://`?

Reply:
I changed the logic.